qa/L0_java_sequence_batcher/SequenceTest.java

```diff
@@ -0,0 +1,638 @@
+
+      for (int idx = 0; idx < output_count[0]; ++idx) {
+        BytePointer cname = new BytePointer((Pointer)null);
+        IntPointer datatype = new IntPointer(1);
+        LongPointer shape = new LongPointer((Pointer)null);
+        LongPointer dim_count = new LongPointer(1);
+        Pointer base = new Pointer();
+        SizeTPointer byte_size = new SizeTPointer(1);
+        IntPointer memory_type = new IntPointer(1);
+        LongPointer memory_type_id = new LongPointer(1);
+        Pointer userp = new Pointer();
+
+        FAIL_IF_ERR(
+            TRITONSERVER_InferenceResponseOutput(
+                response, idx, cname, datatype, shape, dim_count, base,
+                byte_size, memory_type, memory_type_id, userp),
+            "getting output info");
+
+        if (cname.isNull()) {
+          FAIL("unable to get output name");
+        }
+
+        String name = cname.getString();
+        if (!name.equals(output0)) {
+          FAIL("unexpected output '" + name + "'");
+        }
+
+        if ((dim_count.get() != 1) || (shape.get(0) != 1)) {
+          FAIL("unexpected shape for '" + name + "'");
+        }
+
+        if (datatype.get() != expected_datatype) {
+          FAIL(
+              "unexpected datatype '" +
+              TRITONSERVER_DataTypeString(datatype.get()) + "' for '" +
+              name + "'");
+        }
+
+        if (byte_size.get() != expected_byte_size) {
+          FAIL(
+              "unexpected byte-size, expected " +
+              expected_byte_size + ", got " +
+              byte_size.get() + " for " + name);
+        }
+
+        if (memory_type.get() != requested_memory_type) {
+          FAIL(
+              "unexpected memory type, expected to be allocated in " +
+              TRITONSERVER_MemoryTypeString(requested_memory_type) +
+              ", got " + TRITONSERVER_MemoryTypeString(memory_type.get()) +
+              ", id " + memory_type_id.get() + " for " + name);
+        }
+
+        // We make a copy of the data here... which we could avoid for
+        // performance reasons but ok for this sequence example.
+        BytePointer odata = new BytePointer(byte_size.get());
+        output_data.put(name, odata);
+        System.out.println(name + " is stored in system memory");
+        odata.put(base.limit(byte_size.get()));
+      }
+
+      int out = new IntPointer(output_data.get(output0)).get(0);
+      System.out.println("Value: " + out);
+      if(sequence_end){
+        expected_result = GetExpectedResult(model_name, expected_result,
+            input_value, "end");
+        if(out != expected_result){
+          FAIL("Expected result: " + expected_result + ", got " + out);
+        } else {
+          System.out.println(model_name + " test PASSED");
+        }
+      }
+    }
+
+    // Boilerplate main function to run inference
+    // for provided model, custom setting of
+    // sequence metadata
+    public static void
+    main(String[] args) throws Exception
```

Comment:
here too

Reply:
This one does need to be there. It's from the original Simple. We could get rid of it by try-catching the two Thread.sleeps (during server start-up and model readiness) and the CompletableFuture.get() instead. If we do that, it'd be good to do it to our original sample and across all the tests for uniformity (perhaps in the big java-api-prerelease -> main PR).

Reply:
If it's okay, I'll merge for now. If you'd like me to address this across our Simple examples and tests, let me know and I can do so in the big PR ticket or can do so sooner in a separate ticket.